Examples/OneSignalDemo/app/src/main/java/com/onesignal/sdktest/notification/AppRemoteNotificationExtensionService.java

```diff
@@ -0,0 +1,40 @@
+package com.onesignal.sdktest.notification;
+
+import android.content.Context;
+
+import com.onesignal.OSMutableNotification;
+import com.onesignal.OSNotification;
+import com.onesignal.OSNotificationOpenedResult;
+import com.onesignal.OSNotificationReceivedEvent;
+import com.onesignal.OneSignal;
+import com.onesignal.sdktest.R;
+
+public class AppRemoteNotificationExtensionService implements
```

Comment:
Rename class to `AppRemoteNotificationServiceExtension` to match the new `NotificationServiceExtension` naming.